bigquery_etl/query_scheduling/templates/airflow_dag.j2

```diff
@@ -81,6 +85,7 @@ with DAG('{{ name }}', default_args=default_args{%+ if schedule_interval != None
         {% if task_ref.execution_delta -%}
         execution_delta={{ task_ref.execution_delta | format_timedelta | format_repr }},
         {% endif -%}
+        pool='sensor_pool',
```

Comment:
For this to not fail, we'd first need to create this pool. I think it is not *strictly necessary* to fix our issue but potentially a good improvement.